pkg/controller/etcdproxy/helpers.go

```diff
@@ -134,7 +135,8 @@ func newReplicaSet(etcdstorage *etcdstoragev1alpha1.EtcdStorage,
 
 func newService(etcdstorage *etcdstoragev1alpha1.EtcdStorage, etcdControllerNamespace string) *corev1.Service {
 	labels := map[string]string{
-		"controller": "epc",
+		"controller": "etcdproxy",
+		"apiserver":  etcdstorage.Name,
```

Comment:
why do you need the controller label here?  Seems like the apiserver is selective enough.

Reply:
Updated. Not needed at all.